templates/.gitignore

```diff
@@ -11,3 +11,8 @@
 /Gemfile.local.rb
 /coverage
 .byebug_history
+/.env
+/bin/endless-rspec
+/app/assets/javascripts/**/*.js.map
+/app/assets/javascripts/**/*.js
+.tsdrc
```

Comment:
Np xD
On Mar 19, 2016 17:59, "Josua Schmid" <notifications@github.com> wrote:

> In templates/.gitignore
> <https://github.com/renuo/rails-application-setup-guide/pull/2#discussion_r56751617>
> :
>
> > @@ -11,3 +11,8 @@
> >  /Gemfile.local.rb
> >  /coverage
> >  .byebug_history
> > +/.env
> > +/bin/endless-rspec
> > +/app/assets/javascripts/**/*.js.map
> > +/app/assets/javascripts/**/*.js
> > +.tsdrc
>
> sorry ^^
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/renuo/rails-application-setup-guide/pull/2/files/a52c9e3d9e6027d5cd6acbac4eb42b198907a127#r56751617>
>